packages/config/src/handlers/application/applicationConfigHandler.ts

```diff
@@ -40,9 +40,7 @@ export default async function configHandler(
         Promise.all(
           certs.map(async (cert) => ({
             ...cert,
-            svgCode: await getDocumentUrl(cert.svgCode, {
-              Authorization: request.headers.authorization
-            })
+            svgCode: `http://localhost:3040/certificates/${cert.event}.svg`
```

Comment:
@Zangetsu101 this should not be hardcoded as localhost

Reply:
yeah sorry it was my bad. I'll remove it now